### PR TITLE
Release 2.1.2 Version

### PR DESCRIPTION
**Pull request by HarelM**
## Launch Checklist

Bump version number and prepare changelog for next version.
After this is merged a new version will be released.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 ### Features and improvements
 
+- *...Add new stuff here...*
+
+### 🐞 Bug fixes
+
+- *...Add new stuff here...*
```
Comment (wipfli) [inline CHANGELOG.md:9]: Remove below

**Review by wipfli — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (HarelM) [reply CHANGELOG.md:9]: hu?

Comment (wipfli) [reply CHANGELOG.md:9]: Remove `- *...Add new stuff here...*` in v2.1.2 section below.

Comment (HarelM) [reply CHANGELOG.md:9]: Oh, right... I'll wait after the other PR is in to do it.

Comment (HarelM) [reply CHANGELOG.md:9]: Done

**Review by wipfli — APPROVED** [review]

**Pull request merged**